packages/ui/src/components/editor/properties/facer/index.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { useComponent } from '@etherealengine/ecs/src/ComponentFunctions'
+
+import { FaRegFaceFlushed } from 'react-icons/fa6'
+
+import { EntityUUID } from '@etherealengine/ecs'
+import { EditorComponentType, commitProperty } from '@etherealengine/editor/src/components/properties/Util'
+import { FacerComponent } from '@etherealengine/spatial/src/transform/components/FacerComponent'
+import InputGroup from '../../input/Group'
+import NodeInput from '../../input/Node'
+import NodeEditor from '../nodeEditor'
+
+/**
+ * FacerNodeEditor component used to customize the facer element on the scene
+ */
+export const FacerNodeEditor: EditorComponentType = (props) => {
+  const { t } = useTranslation()
+
+  const facerComponent = useComponent(props.entity, FacerComponent)
+
+  return (
+    <NodeEditor
+      entity={props.entity}
+      component={FacerComponent}
+      name={t('editor:properties.facer.name')}
+      description={t('editor:properties.facer.description')}
+    >
+      <InputGroup name="Target" label={t('editor:properties.facer.target')}>
```

Comment:
there is an example in RenderSettingsNodeEditor

```ts
  const directionalLightOptions = [
    {
      label: 'None',
      value: '' as EntityUUID
    }
  ].concat(
    useQuery([DirectionalLightComponent]).map((entity) => {
      return {
        label: getComponent(entity, NameComponent),
        value: getComponent(entity, UUIDComponent)
      }
    })
  )```

It's for getting a list of options with the name of entity and its uuid automatically tied to it. Could be a better option for this input as it's more usable.

Reply:
I'm wanting to avoid that type of selection since it gets confusing with large scenes (this dropdown in this context would include every single entity with a transform component in the entire scene). Trying to follow Unreal + Unity in allowing drag and drop as the primary method of setting scene objects as inputs

Reply:
Sounds good